src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+import duke.util.DukeException;
+
+import java.util.Hashtable;
+import java.util.Scanner;
+import java.util.function.Consumer;
+
+public class Duke {
+
+    private static TaskBank taskBank = new TaskBank("data/duke.txt");
+
+    private static Hashtable<String, Consumer<String>> initCommandTable() {
+
+        Hashtable<String, Consumer<String>> ret = new Hashtable<>();
+
+        ret.put("list", (x) -> Ui.printTasks(taskBank.getTasks()));
+        ret.put("event", (x) -> taskBank.addTask(x, Event::create));
+        ret.put("deadline", (x) -> taskBank.addTask(x, Deadline::create));
+        ret.put("todo", (x) -> taskBank.addTask(x, ToDo::create));
+        ret.put("done", (x) -> taskBank.markTask(x));
+        ret.put("delete", (x) -> taskBank.deleteTask(x));
+        ret.put("find", (x) -> UI.printTasks(taskBank.searchTasks(x)));
+
+        return ret;
+    }
+
+    public static void main(String[] args) {
+
+        Ui.welcomeMessage();
+        Ui.printDivider();
+
+        Scanner sc=new Scanner(System.in);
```

Comment:
Could u add some spaces here, so as to be consistent with the coding standard.
```suggestion
        Scanner sc = new Scanner(System.in);
```